br/pkg/task/stream.go

```diff
@@ -1291,39 +1306,39 @@ func countIndices(ts map[int64]*metautil.Table) int64 {
 func getLogRange(
 	ctx context.Context,
 	cfg *Config,
-) (uint64, uint64, error) {
+) (uint64, uint64, uint64, error) {
```

Comment:
The number of return values is getting increase. Can we define a structure for it?